ansiblelint/rules/ShellModuleScalarRule.py

```diff
@@ -0,0 +1,16 @@
+from ansiblelint import AnsibleLintRule
+
+class ShellModuleScalarRule(AnsibleLintRule):
+  id = '700'
+  shortdesc = 'Scalar(">") is missing in shell command'
+  description = 'Required scalar if shell commands are greater than three'
```

Comment:
Use scalar for shell commands with multiple args

Reply:
Done

Reply:
Requires scalar if shell command consist of args greater than three

Reply:
done